src/componenets/header.js

```diff
@@ -0,0 +1,61 @@
+class foodAppHeader extends HTMLElement {
+  constructor() {
+    super();
+
+    this.links = [
+      {
+        name: 'HOME',
+        href: 'index.html',
+        active: this.checkPathname('index.html'),
+      },
+      {
+        name: 'SHOPPING LIST',
+        href: 'shoppingList.html',
+        active: this.checkPathname('shoppingList.html'),
+      },
+      {
+        name: 'WINE',
+        href: 'page.html',
+        active: this.checkPathname('p.html'),
+      },
+      {
+        name: 'NUTRITION',
+        href: 'page.html',
+        active: this.checkPathname('p.html'),
+      },
+      {
+        name: 'FAST FOOD',
+        href: 'page.html',
+        active: this.checkPathname('p.html'),
+      },
+    ];
+  }
+
+  checkPathname(path) {
+    const pathname = window.location.pathname.split('/');
+    return pathname[1] === path;
```

Comment:
magic number, variable with some good name would be better, when you get back to this code you will not have to think what this 1 is